### PR TITLE
Refactor insights management control card

### PR DESCRIPTION
**Pull request by planarvoid**
In order to add the Insights management introduction news card we first need to refactor the current structure with the existing "Control" card. Originally I had one big PR but I've decided to split this part to make everything more readable and easier to review.

To test - there shouldn't be any functionality changes:
* Go to Stats/Insights
* Scroll down
* The "Edit" button is still on the bottom of the screen
* Click on the "Edit" button
* You get redirected to the "Insights management"

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Review by khaykov — APPROVED** [review]
Looks good to me! Feel free to merge it after the conflict is resolved :)

**Comment by planarvoid** [comment]
@khaykov thanks for the review! Do you think you can check the FluxC PR before I merge this? I forgot to assign you before - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1265

**Comment by khaykov** [comment]
@planarvoid Flux-C PR looks good! We can merge this one after conflicts are fixed and Flux-C reference points to the latest commit.

**Pull request merged**
